Moonshot/procedural_map_generation/BaddieBuilder.gd

```diff
@@ -0,0 +1,36 @@
+extends Node
+class_name BaddieBuilder
+
+# public modifyable attributes on a baddie
+const attr: Dictionary = {
+	"INITIAL_HP": "inital_hp", # float
+	"HP": "hp", # float
+	"GRAVITY": "gravity", # int
+	"SPEED": "speed", # int
+	"VELOCITY": "velocity", #  Vector2
+	"SHOT_SPEED": "shot_speed", # int
+	"SHOT_DAMAGE": "shot_damage", # float
+	"ITEM_DROP": "item_drop" # Object
+}
```

Comment:
Not a criticism, when I start building the difficulty cotroller today I will probs change these to 'min' and 'max' values that are randomly setwhen spawning a baddie. Could go further and set them as distributions but i thinkthat wouldbeoverkill.

Reply:
Oh wait, my mistake, this doesn't spawn baddies, it is just a wrapper for one 👍